.github/workflows/annotated-pkgs.yml

```diff
@@ -24,9 +24,9 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
```

Comment:
This is causing tests to fail due to the following issue: https://github.com/actions/setup-node/issues/317